reference/forms/types/options/error_mapping.rst.inc

```diff
@@ -34,7 +34,14 @@ Here are the rules for the left and the right side of the mapping:
   object, the property path is ``[indexName]``;
 * You can construct nested property paths by concatenating them, separating
   properties by dots. For example: ``addresses[work].matchingCityAndZipCode``;
-* The left side of the error mapping also accepts a dot ``.``, which refers
-  to the field itself. That means that any error added to the field is added
-  to the given nested field instead;
 * The right side contains simply the names of fields in the form.
+
+Additionally, you can set the left side to dot (``.``), which refers to any
+unmapped property or method needing validation to the given nested field instead
+of bubbling them to the form::
```

Comment:
@WouterJ, thanks for merging this. I agree with @xabbuh for the rephrasing except for "parent form" and "a particular field" because it's not obvious that `(.)` symbolise the current FormType being configured with `defaultOptions()|configureOptions()`. It should be respectively "current FormType"  and "one of its child"  instead IMHO.